specs/posix/yum_sources.table

```diff
@@ -0,0 +1,9 @@
+table_name("yum_sources")
+description("Current list of APT repositories or software channels.")
```

Comment:
I think this should be updated to be "Current list of Yum repositories or software channels".